cmd/nerdctl/container/container_top.go

```diff
@@ -66,9 +67,18 @@ func topAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 	defer cancel()
-	return container.Top(ctx, client, args, types.ContainerTopOptions{
+
+	containerID := args[0]
```

Comment:
Please check len to avoid out-of-range panic

Reply:
minNArgs should cover it I think

Reply:
Yes, cobra.MinimumNArgs(1) should prevent this from becoming out of range